packages/template-retail-react-app/app/components/item-variant/item-price.jsx

```diff
@@ -66,7 +66,11 @@ const ItemPrice = ({currency, align = 'right', baseDirection = 'column', ...prop
         >
             {basePrice && price !== basePrice && (
                 <HideOnDesktop>
-                    <PricePerItem currency={currency} basePrice={basePrice} basket={basket} />
+                    <PricePerItem
+                        currency={currency}
+                        basePrice={basePrice.toString()}
```

Comment:
Looks like this is ultimately passed through to `FormattedNumber` - should it stay a number and the change here be to the `PropTypes`?